foundry.toml

```diff
@@ -5,6 +5,7 @@ libs = ['lib']
 test = 'test/foundry'
 cache_path = 'forge-cache'
 script = 'scripts'
+evm_version = 'shanghai'
```

Comment:
this will brick us on non-push0 chains. Optimism and arbitrum have push0.  superchains like mode and fraxtal should.  Mantle and most other chains should also have this but it still prob doesn't make sense to set to shanghai

Reply:
ramses' deployed contract bytecode didnt have push0, throwing errors